README.md

```diff
@@ -28,6 +28,9 @@ Below is a curated list of awesome digital tools for activism. These are tools f
  * [TakeCharge.io](https://github.com/controlshift/prague-server) - opensource or SaaS online donations powered by Stripe.
  * [VictoryKit](http://www.victorykitapp.com) - Optimize petition campaign content. Not actively maintained.
  * [Countdown](https://github.com/drewrwilson/countdown) - A quick & simple tool for making your own webpage with a countdown to your upcoming event.
+ * [DemocracyOS](http://democracyos.org) - An open-source platform for public policy engagement, open feedback, and participatory governance. 
+ * [Madison](http://mymadison.io) - An open-source platform for policy proposals and petitions to be uploaded for robust public feedback and exchange.
```

Comment:
Looks like this is now retired. Looks awesome tho! 

![image](https://user-images.githubusercontent.com/4324656/55023160-926f3980-4fd2-11e9-85d9-b469d727fedc.png)